specification/maps/data-plane/Common/preview/1.0/common.json

```diff
@@ -1,712 +1,712 @@
+            }
+          ]
+        },
+        "x-ms-parameter-location": "method"
+      },
+      "TextFormat": {
+        "name": "format",
+        "description": "Desired format of the response. Value can be either _json_ or _xml_.",
+        "type": "string",
+        "in": "path",
+        "required": true,
+        "enum": [
+          "json",
+          "xml"
+        ],
+        "x-ms-enum": {
+          "name": "TextFormat",
+          "modelAsString": true,
+          "values": [
+            {
+              "value": "json",
+              "description": "[The JavaScript Object Notation Data Interchange Format](https://tools.ietf.org/html/rfc8259)"
+            },
+            {
+              "value": "xml",
+              "description": "[The Extensible Markup Language](https://www.w3.org/TR/xml/)"
+            }
+          ]
+        },
+        "x-ms-parameter-location": "method"
+      },
+      "GeographicResourceLocation": {
+        "name": "geography",
+        "description": "This parameter specifies where the Azure Maps Creator resource is located.  Valid values are us and eu.",
+        "in": "path",
+        "required": true,
+        "type": "string",
+        "default": "us",
+        "enum": [
+          "us",
+          "eu"
+        ],
+        "x-ms-enum": {
+          "name": "GeographicResourceLocation",
+          "modelAsString": true,
+          "values": [
+            {
+              "value": "us",
+              "description": "Used to access an Azure Maps Creator resource in the United States"
+            },
+            {
+              "value": "eu",
+              "description": "Used to access an Azure Maps Creator resource in Europe"
+            }
+          ]
+        },
+        "x-ms-parameter-location": "client"
+      },
+      "View": {
+        "name": "view",
+        "in": "query",
+        "description": "The View parameter specifies which set of geopolitically disputed content is returned via Azure Maps services, including  borders and labels displayed on the map. The View parameter (also referred to as “user region parameter”) will show the  correct maps for that country/region. By default, the View parameter is set to “Unified” even if you haven’t defined it in  the request. It is your responsibility to determine the location of your users, and then set the View parameter correctly  for that location. Alternatively, you have the option to set ‘View=Auto’, which will return the map data based on the IP  address of the request. The View parameter in Azure Maps must be used in compliance with applicable laws, including those  regarding mapping, of the country where maps, images and other data and third party content that you are authorized to  access via Azure Maps is made available. Example: view=IN.\n\nPlease refer to [Supported Views](https://aka.ms/AzureMapsLocalizationViews) for details and to see the available Views.",
```

Comment:
This is an extremely vague name that doesn't even imply what it actually does. I would recommend using `x-ms-client-name` to create more descriptive name .